packages/react-server/src/ping/ping.test.ts

```diff
@@ -0,0 +1,16 @@
+import {saddle, unsaddle} from 'saddle-up/koa';
+import {StatusCode} from '@shopify/network';
+
+import {ping} from './ping';
+
+describe('ping', () => {
+  afterAll(unsaddle);
+
+  it('responds with pong', async () => {
```

Comment:
I switched this test to show that we can write really nice little tests for simple middleware without feeling like overkill